exporter/clickhousetracesexporter/migrations/000023_span_attributes_mv.up.sql

```diff
@@ -7,7 +7,7 @@ CREATE TABLE IF NOT EXISTS signoz_traces.span_attributes ON CLUSTER cluster (
     float64TagValue Nullable(Float64) CODEC(ZSTD(1)),
     isColumn bool CODEC(ZSTD(1)),
 ) ENGINE ReplacingMergeTree
-ORDER BY (tagKey, tagType, dataType, stringTagValue, float64TagValue)
+ORDER BY (tagKey, tagType, dataType, stringTagValue, float64TagValue, isColumn)
```

Comment:
Why is this needed here?

Reply:
It's possible that there are two keys with the same name but one is column and the other is not, as we discussed we don't want to put restrictions on insertion of such keys and have to handle them on query builder (while querying)

Reply:
I think this will cause issues with materialising the column in future.